include/xten/Dialect/XTenNN/IR/XTenNNOps.td

```diff
@@ -181,4 +181,26 @@ def XTenNN_LoadExternalConstOp: XTenNN_Op<"load_external_const", [
   let assemblyFormat = [{ attr-dict `->` type($output) }];
 }
 
+//===----------------------------------------------------------------------===//
+// Ops that are missing from the TOSA standard
+//===----------------------------------------------------------------------===//
+
+def TosaExtension : NativeOpTrait<"TosaExtension">;
+def ElementwiseUnary : NativeOpTrait<"ElementwiseUnary">;
+
+def XtenNN_UnifiedSignOp: XTenNN_Op<"unified.sign", [Pure, TosaExtension, ElementwiseUnary, SameOperandsAndResultElementType]> {
```

Comment:
What does `unified` mean here?

Reply:
Because it is unified from onnx and torch. Naming is hard....

Reply:
I feel like `xten_nn.sign` is good enough

Reply:
But it is not the same class of op like the subgraph for example. I feel like some kind of separation needs to happen here. I even thought about a new dialect